lang/en.json

```diff
@@ -157,6 +157,10 @@
         "label": "Activation Condition",
         "hint": "Condition required to activate this activity."
       },
+      "override": {
+        "label": "Override Activation",
+        "hint": "Override the activation information from the item with values for this activity."
```

Comment:
This is confusingly worded, does it override the values _on the activity_ with values from the item, or the other way around

Reply:
It's the other way around, but I agree the wording might be difficult to parse for the average user. I think we could probably do with simplifying the language across the board on a lot of our forms.

For this one, maybe something like: `"Use these values instead of the item's when using this activity."`